Zilon.Core/Zilon.Core/Tactics/TacticalActUsageService.cs

```diff
@@ -127,7 +127,8 @@ private void UseAct(IActor actor, IAttackTarget target, ITacticalAct act)
 
             if (!isInDistance)
             {
-                throw new UsageOutOfDistanceException("Попытка атаковать цель, находящуюся за пределами атаки.");
+                //throw new UsageOutOfDistanceException("Попытка атаковать цель, находящуюся за пределами атаки.");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5544638)